client/keys/export.go

```diff
@@ -25,13 +29,31 @@ func ExportKeyCommand() *cobra.Command {
 				return err
 			}
 
-			armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
-			if err != nil {
-				return err
+			hex, _ := cmd.Flags().GetBool(flagHex)
+
+			if !hex {
+				armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
+				if err != nil {
+					return err
+				}
+
+				cmd.Println(armored)
+				return nil
+			} else {
+				hexPrivKey, err := clientCtx.Keyring.ExportPrivKeyHex(args[0])
```

Comment:
As I said before, this may risk exposing users' private key material in clear text. At the very least we need a confirmation prompt here...

Reply:
Added confirmation prompt